### ISSUE TITLE
ConfigError raised when used with django-polymorphic

### ISSUE THREAD

**Issue by jackton1** [ericls/django-hashids#19]
https://github.com/ericls/django-hashids/blob/master/django_hashids/field.py#L50-L57

[django-polymorphic](https://github.com/django-polymorphic/django-polymorphic) or via [model inheritance](https://docs.djangoproject.com/en/3.2/topics/db/models/#model-inheritance)

**Comment by ericls** [comment]
Taking a look

**Comment by ericls** [comment]
I can see how this can be a problem but can't reproduce the issue on my end.
This is what I'm doing:

``` python3
class TestBase(Model):
    class Meta:
        abstract = True

    hashid =  HashidsField()
    # This also works
    # hashid =  HashidsField(hashids_instance=Hashids(salt="123"))


class InheritanceModel(TestBase):
    pass

class InheritanceModel2(TestBase):
    pass
```

And I'm able to test it with success:
```python3
def test_model_inheritance():
    from tests.test_app.models import InheritanceModel, InheritanceModel2

    instance = InheritanceModel.objects.create()
    InheritanceModel.objects.filter(hashid=instance.hashid).get() == instance

    instance2 = InheritanceModel2.objects.create()
    InheritanceModel2.objects.filter(hashid=instance2.hashid).get() == instance2
```

Could you please provide some reproducing steps?

**Comment by jackton1** [comment]
Sure, I saw the issue with django-polymorphic if that helps. I’ll try to setup a test to replicate the issue.

**Comment by ericls** [comment]
Got it. I'm going to give it a try as well.

**Comment by ericls** [comment]
Hi @jackton1, I think #25 is related to this and #26 might fix it. I still hasn't being able to reproduce this issue but would you like to give the new patch a try?

**Comment by ericls** [comment]
Should be fixed in version 0.6.0(11e25d02779d41e16c72dea0a7c58e1261b2b41f)
